### ISSUE TITLE
mounting developer images should be done using major version

### ISSUE THREAD

**Issue by vdelendik** [zebrunner/appium#43]
make sure to cut platformVersion before mounting developr images.
it is impossible to find 15.2.1 etc as 15.2 only available on Apple side
```
major_device_version=$(echo "$DEVICE_OS_VERSION" | cut -f1,2 -d '.')
```

**Comment by vdelendik** [comment]
fixed. to test find any iOS devices with patched iOS version, for example 14.7.1, 15.2.1 etc and connect to the cloud.
inspect appium log to see that mounting developer images should be done using major version, i.e. 14.7 and 15.2

**Comment by vdelendik** [comment]
verified on my own
